paper/paper.md

```diff
@@ -41,18 +41,18 @@ TO DO
 # HyP3 Description
 
 ## Development History
-HyP3 grew out of a need for a standardized pipeline to perform image processing workflows at the Alaska Satellite Facility (ASF) Distributed Active Archive Center (DAAC). HyP3 was initially created in 2017 by a group of undergraduate University of Alaska Fairbanks students interning at ASF. These students created a prototype version of HyP3 that was designed to handle workflows that ran on-premise and in the AWS cloud. This prototype was iterated upon for several more years, and by 2019 it was capable of performing many SAR-specific image processing workflows.
+HyP3 grew out of a need for a standardized pipeline to perform image processing workflows at the Alaska Satellite Facility (ASF) Distributed Active Archive Center (DAAC). HyP3 was initially created in 2016 by a group of undergraduate students interning at ASF. These students created a prototype version of HyP3 that was designed to handle workflows running on-premises and in the AWS cloud. This prototype was iterated upon for several more years, and by 2019 it was capable of performing many SAR-specific image processing workflows.
```

Comment:
https://www.govloop.com/on-premise-vs-on-premises-the-debate-and-resolution/
Hot topic (I had no idea, and it's very fascinating!). If we go with the approach outlined in this article, we should probably un-hyphenate, too: 
"handle workflows running on premises".

Reply:
I didn't realize this was even a debate, I just assumed that the [Wikipedia article](https://en.wikipedia.org/wiki/On-premises_software) probably knew best.